zerver/lib/test_classes.py

```diff
@@ -532,6 +532,9 @@ def client_post(
                 encoded = urlencode(info, doseq=True)
             else:
                 content_type = MULTIPART_CONTENT
+        if content_type.startswith("multipart/form-data"):
```

Comment:
Should this be `elif`? Just for readability to make the control flow more obvious.